Makefile

```diff
@@ -6,7 +6,11 @@ build:
 	go build -ldflags "-X 'github.com/conduitio-labs/conduit-connector-mongo.version=${VERSION}'" -o conduit-connector-mongo cmd/connector/main.go
 
 test:
-	go test $(GOTEST_FLAGS) ./...
+	docker run --rm -d -p 27017:27017 --name mongodb mongo --replSet=test && \
+	sleep 4 && \
```

Comment:
Do you think 4 is enough? Is it possible that on old machines it is need take more?

Reply:
Yeah, I think it's possible that it may take longer on an old machine...
I can make it configurable. For example, via an environment variable, `MONGODB_START_TIMEOUT`. With the default set to `4`.
What do you think?

Reply:
But, by the way, even on GitHub CI, it manages to start up in 4 seconds.